packages/cli/package.json

```diff
@@ -27,7 +27,10 @@
   ],
   "type": "module",
   "exports": "./dist/index.js",
-  "bin": "./dist/index.js",
+  "bin": {
+    "shadcn-ui": "./dist/index.js",
+    "shui": "./dist/index.js"
```

Comment:
Do you really think is necessary add this command to avoid only 4 letters? In my opinion is too much

Reply:
Not necessary at all. It's just a QoL improvement, especially when invoking the command numerous times given that one of those four characters is a hyphen.

Reply:
`ui` would be a better alias

Reply:
`ui` is better and it shouldn't conflict with npm's `ui` package.